SA3/qa_web/views.py

```diff
@@ -470,5 +470,22 @@ def delete(request, id_):
     return HttpResponseRedirect('/QuestionIndex')
 
 
+# edit Answers
+@login_required(login_url='/login/')
+def editAnswers(request, id_, a_id):
+
+    a = get_object_or_404(Answers, pk=a_id)
+    if a.owner != request.user:
+        return HttpResponseForbidden()
+
+    form = EditForm(request.POST)
+    if request.POST and form.is_valid():
+        a.content = request.POST['content']
+        a.owner = request.user
+        a.save()
+        return HttpResponseRedirect('/questions/{id}/'.format(id=id_))
+    return render(request,'qa_web/edit.html', context={'Post':a, 'isAnswer': True})
```

Comment:
Once again here, can `isAnswer` be `is_answer'. This would have to change in your views too.